commands/import.go

```diff
@@ -146,28 +161,37 @@ func setRolling(dbExists bool, dbIsRolling bool, dbCurrChunk int, dbTotalChunks
 		}
 	}
 
+	//set cfg.CurrentChunk
 	if userCurrChunk != -1 { // user gave the current chunk via command line
 		// use the user-provided value
 		cfg.CurrentChunk = userCurrChunk
 	} else { // user didn't specify the current chunk
 		if !dbExists {
-			// if the databse doesn't exist, then assume this is the first chunk
+			// if the database doesn't exist then assume this is the first chunk
+			cfg.CurrentChunk = 0
+		} else if deleteOldData && dbExists && !dbIsRolling {
```

Comment:
This covers the cases of `rita import --delete --rolling` and `rita import --delete` running against a non-rolling database.